### ISSUE TITLE
Setting color to be transparent also makes dash_background_color transparent

### ISSUE THREAD

**Issue by rwrx** [tangrams/tangram#741]
I would like to kindly ask you a question regarding color transparency. I have this use case:

The first color `track_background_color` is set as transparent. The second color `track_color` is not transparent, this color is used in style `dashed_grade5` as `dash_background_color`. When this style is applied, all lines are transparent although the `dash_background_color` is not set as transparent and we want to have it opaque. Should it be like this or is it an error which can be fixed? Here are snippets from actual scene file:

```
global:
    ...
    track_background_color: [1.0, 1.0, 1.0, 0.3]
    track_color: [0.6, 0.4, 0.0]
    ...
   
styles:
    ...
    dashed_grade5:
        base: lines
        blend: translucent
        dash: [1.5, 2.5]
        dash_background_color: global.track_color
        material:
            diffuse: 1.0
            ambient: 1.0
    ...
   
layers:
    ...
    roads:
        ...
        grade5:
            filter: { tracktype: grade5 }
            draw:
                lines:
                    style: dashed_grade5
                    color: global.track_background_color
                    width: [ [13, 1px], [16, 3px]]
                    visible: true
                    outline:
                        color: global.track_background_color
                        dash_background_color: global.track_background_color
                        width: 5px
```

![dashed](https://user-images.githubusercontent.com/2378743/72020571-5d378d80-326c-11ea-9699-c9e3eae95420.png)

**Comment by bcamper** [comment]
Hello, thanks for your example styling -- I do think this is a bug. Based on your description and style, I think this is what you would have expected, can you confirm?

![tangram-@17 8875 37 7936 -122 4628_2020-01-10_00 54 50](https://user-images.githubusercontent.com/16733/72209008-f8e91980-3477-11ea-8dfd-8f5923822b6f.png)

**Comment by rwrx** [comment]
Yes, this is what I expected to see with that styling. I am also not sure if this is also present in Tangram-ES.

**Comment by bcamper** [comment]
Thanks, I've opened the #742 PR to fix this (and related logic issues with dash/line texture/blend mode behavior). This issue **does** appear to exist in Tangram ES as well, we will take care of opening a corresponding issue there.

**Comment by rwrx** [comment]
Thank you a lot.

**Comment by bcamper** [comment]
Fix released in [v0.20.1](https://github.com/tangrams/tangram/releases/tag/v0.20.1)
